jetty-http2/http2-http-client-transport/src/main/java/org/eclipse/jetty/http2/client/http/HttpClientTransportOverHTTP2.java

```diff
@@ -60,7 +60,7 @@ public HttpClientTransportOverHTTP2(HTTP2Client client)
         setConnectionPoolFactory(destination ->
         {
             HttpClient httpClient = getHttpClient();
-            return new MultiplexConnectionPool(destination, httpClient.getMaxConnectionsPerDestination(), destination, httpClient.getMaxRequestsQueuedPerDestination());
+            return new MultiplexConnectionPool(destination, httpClient.getMaxConnectionsPerDestination(), destination, 1);
```

Comment:
I'm assuming the 1 is there until overridden by the settings frame?   Eitherway a comment would be good explaining why 1.

Reply:
Done.